src/client/graphics/layers/EventsDisplay.ts

```
@@ -435,12 +437,35 @@ export class EventsDisplay extends LitElement implements Layer {
+          action: () => this.eventBus.emit(new GoToPlayerEvent(traitor)),
+          preventClose: true,
+        },
+      ];
+      // Show dismiss alert button if alert frame is enabled
+      if (this.userSettings.alertFrame()) {
+        buttons.push({
+          text: "Dismiss alert",
+          className: "btn-info",
+          action: () => {
+            const alertFrame = document.querySelector("alert-frame") as any;
+            if (alertFrame && alertFrame.dismissAlert) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)